### DISCUSSION TITLE
Download playlist for jellyfin and use playlist order?

### DISCUSSION THREAD

**Discussion by Plutoaurus** [jmbannon/ytdl-sub#1109] [Q&A] [answered]
Hi. 
I have a few playlists that contain videos from a couple of channels in each playlist. 

I would like to download each playlist for use in jellyfin and have the episode order as in the playlist. 

I've read through this
[](https://ytdl-sub.readthedocs.io/en/latest/prebuilt_presets/tv_shows.html#tv-show-collection)

And it looks like I need

jellyfin_tv_show_collection
season_by_collection__episode_by_playlist_index

But can't work out where that needs to go.  Can't find any examples either. 

Please could anyone give me a quick example of where to put this?

Thanks

**Comment by jmbannon** [comment, marked as answer]
Make a preset in a config file:

```
presets:
  Playlist By Index:
    preset:
      - "jellyfin_tv_show_collection"
      - "season_by_collection__episode_by_playlist_index"
```
Then use that in your subs:
```
Playlist By Index:
  "~My Playlist Show":
    s01_url: "https://...."
```
Since it uses TV show collection, you need to use that syntax.


Made https://github.com/jmbannon/ytdl-sub/issues/1111 to remind myself to add documentation

**Reply by Plutoaurus** [reply]
Thank you, il give that a go.
